src/main/java/it/gov/pagopa/payhub/activities/service/debtposition/ionotification/IONotificationService.java

```diff
@@ -0,0 +1,112 @@
+import it.gov.pagopa.pu.ionotification.dto.generated.NotificationRequestDTO;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static it.gov.pagopa.pu.ionotification.dto.generated.NotificationRequestDTO.OperationTypeEnum.CREATE_DP;
+
+@Lazy
+@Service
+@Slf4j
+public class IONotificationService {
+
+    private final DebtPositionTypeOrgService debtPositionTypeOrgService;
+    private final IONotificationFacadeService ioNotificationFacadeService;
+    private final NotificationRequestMapper notificationRequestMapper;
+    private final InstallmentOperationTypeResolver installmentOperationTypeResolver;
+
+    public IONotificationService(DebtPositionTypeOrgService debtPositionTypeOrgService, IONotificationFacadeService ioNotificationFacadeService, NotificationRequestMapper notificationRequestMapper, InstallmentOperationTypeResolver installmentOperationTypeResolver) {
+        this.debtPositionTypeOrgService = debtPositionTypeOrgService;
+        this.ioNotificationFacadeService = ioNotificationFacadeService;
+        this.notificationRequestMapper = notificationRequestMapper;
+        this.installmentOperationTypeResolver = installmentOperationTypeResolver;
+    }
+
+    public List<MessageResponseDTO> sendMessage(DebtPositionDTO debtPositionDTO, Map<String, IupdSyncStatusUpdateDTO> iupdSyncStatusUpdateDTOMap) {
+        List<MessageResponseDTO> response = new ArrayList<>();
+
+        DebtPositionDTO filteredDebtPosition = filterInstallmentsByIUD(debtPositionDTO, iupdSyncStatusUpdateDTOMap);
+
+        if (!filteredDebtPosition.getPaymentOptions().isEmpty()) {
+            DebtPositionDTO createDpDebtPosition = calculateOperationType(filteredDebtPosition);
+
+            if (!createDpDebtPosition.getPaymentOptions().isEmpty()) {
+                IONotificationDTO ioNotificationDTO = debtPositionTypeOrgService
+                        .getIONotificationDetails(debtPositionDTO.getDebtPositionTypeOrgId(), CREATE_DP);
+
+                if (ioNotificationDTO != null) {
+                    response = processNotifications(createDpDebtPosition, ioNotificationDTO);
+                }
+            }
+        }
+
+        return response;
+    }
+
+    /**
+     * Maps and Sends notifications for the given debt position.
+     */
+    private List<MessageResponseDTO> processNotifications(DebtPositionDTO debtPositionDTO, IONotificationDTO ioNotificationDTO) {
+        List<NotificationRequestDTO> notifications = notificationRequestMapper.map(
+                debtPositionDTO, ioNotificationDTO, CREATE_DP
+        );
+
+        return notifications.stream()
+                .map(ioNotificationFacadeService::sendMessage)
+                .toList();
+    }
+
+    /**
+     * Filters payment options to keep only installments with the CREATE_DP operation type.
```

Comment:
this is wrong... it should no be the Activity to decide if notify or not just the CREATE.
it should the DebtPositionTypeOrg configuration to decide.
you should call the DebtPositionTypeOrg with the calculated operationType: if it return a config, you have to send the message, otherwise not